examples/fic-ecl-connection/main.tf

```
@@ -0,0 +1,54 @@
+provider "fic" {
+    auth_url          = var.fic_auth_url
+    user_name         = var.fic_api_key
+    password          = var.fic_api_secret
+    tenant_id         = var.fic_tenant_id
+    user_domain_id    = "default"
+    project_domain_id = "default"
+}
+
+provider "ecl" {
+    auth_url          = var.ecl_auth_url
+    user_name         = var.ecl_api_key
+    password          = var.ecl_api_secret
+    tenant_id         = var.ecl_tenant_id
+    user_domain_id    = "default"
+    project_domain_id = "default"
+}
+
+module "fic-router" {
+    source  = "./modules/fic-router"
```

```suggestion
    source  = "./fic-router"
```
`modules` という階層はあえて設ける必要はないと思います。